packages/forms/backend/sirius-components-widget-reference/src/main/resources/schema/reference.graphqls

```diff
@@ -7,6 +7,7 @@ type ReferenceWidget implements Widget {
   readOnly: Boolean!
   reference: Reference!
   referenceValues: [ReferenceValue!]!
+  referenceOptions: [ReferenceValue!]!
```

Comment:
Is `referenceOptions` mandatory in case of a mono-valued reference?

Reply:
Yes, we use now the same autocomplete component for both case (mono and multi valued)